wis2box-broker/entrypoint.sh

```diff
@@ -16,6 +16,7 @@ echo "Setting mosquitto authentication"
 if [ ! -e "/mosquitto/config/password.txt" ]; then
     echo "Adding wis2box users to mosquitto password file"
     mosquitto_passwd -b -c /mosquitto/config/password.txt $WIS2BOX_BROKER_USERNAME $WIS2BOX_BROKER_PASSWORD
+    mosquitto_passwd -b /mosquitto/config/password.txt $WIS2BOX_CAP_USERNAME $WIS2BOX_CAP_PASSWORD
```

Comment:
Propose the following:

- assume "extra" broker usernames/passwords are formulated like `WIS2BOX_BROKER_USERNAME_<NAME>` and `WIS2BOX_BROKER_PASSWORD_<NAME>`
- assume "extra" broker authenticated topics are formulated like `WIS2BOX_BROKER_TOPIC_<NAME>`

We can introduce a final block like:

```bash
for i in `env | grep -Ee "\<WIS2BOX_BROKER_USERNAME_[[:alnum:]]+"`; do
    NAME_TAIL=`echo $i | awk -FWIS2BOX_BROKER_USERNAME_ '{print $2}' | awk -F= '{print $1}'`
    username=WIS2BOX_BROKER_USERNAME_$NAME_TAIL
    password=WIS2BOX_BROKER_PASSWORD_$NAME_TAIL
    topic=WIS2BOX_BROKER_TOPIC_$NAME_TAIL
    echo ${!username}, ${!password}
    mosquitto_passwd -b /mosquitto/config/password.txt ${!username} ${!password}
    echo "user ${!username}" >>  /mosquitto/config/acl.conf
    echo "topic readwrite ${!topic}" >>  /mosquitto/config/acl.conf
done
```